syz-ci/jobs.go

```diff
@@ -527,7 +528,7 @@ func (jp *JobProcessor) testPatch(job *Job, mgrcfg *mgrconfig.Config) error {
 	}
 
 	log.Logf(0, "job: building kernel...")
-	kernelConfig, _, err := env.BuildKernel(mgr.mgrcfg.Compiler, mgr.mgrcfg.Userspace, mgr.mgrcfg.KernelCmdline,
+	kernelConfig, _, err := env.BuildKernel(mgr.mgrcfg.Compiler, "", mgr.mgrcfg.Userspace, mgr.mgrcfg.KernelCmdline,
```

Comment:
> I think this should also use cache, because why not? Should speed up patch testing requests. Or there are some issue with this? If so, please add a comment.

> And Manager.build in manager.go would also benefit from use of ccache.

These are both fixed.

Reply:
I don't see this is done. Did you forget to push?

Reply:
I pushed now updated branch. Testing it took a bit longer than I thought. Sorry for inconvenience.